packages/cli/src/ui/hooks/shellCommandProcessor.test.ts

```diff
@@ -208,46 +219,6 @@ describe('useShellCommandProcessor', () => {
       vi.useRealTimers();
     });
 
-    it('should throttle pending UI updates for text streams', async () => {
```

Comment:
nit: is removing this intentional? seems we should still have it for the non interactive cases?